src/esockd.appup.src

```diff
@@ -1,21 +1,30 @@
 %%-*-: erlang -*-
 
-{"5.8.3",
-  [{<<"5.8.[0-2]">>,
+{VSN,
```

Comment:
This is our thing. Will cause other users to fail.
We haven used this feature only in emqx plugins.

Reply:
I just feel very urgly to hardcode version number in both app src and appup src. 

isnt this appup.src file for emqx only? rebar3's appup plugin use another template. OTP use yet another one?

neither of them use hardcoded vsn number.

if we hardcode, what is the difference between appup and appup.src?